packages/formvuelate/src/SchemaForm.vue

```diff
@@ -68,7 +70,8 @@ export default {
     useCustomFormWrapper: {
       type: Boolean,
       default: false
-    }
+    },
+    debug: { type: Boolean, default: false }
```

Comment:
Maybe for the future but we could drop this in non-es/unminified bundles with compilation flags to avoid people leaking this to their production.